scripts/test.sh

```diff
@@ -1,106 +1,100 @@
-#!/bin/bash -e
-# The script does automatic checking on a Go package and its sub-packages, including:
-#   - gofmt         (https://golang.org/cmd/gofmt)
-#   - goimports     (https://godoc.org/cmd/goimports)
-#   - go vet        (https://golang.org/cmd/vet)
-#   - test coverage (https://blog.golang.org/cover)
-
-COVERALLS_TOKEN=t47LG6BQsfLwb9WxB56hXUezvwpED6D11
-COVERALLS_MAX_ATTEMPTS=5
-TEST_DIRS="main.go mesos/"
-PKG_DIRS=". ./mesos/..."
-IGNORE_PKGS="mesos_pb2"
```

Comment:
We don't want to test the `mesos_pb2` package created by `make protobuf`; it's generated code.